### PR TITLE
Add graphql-language-service-cli

### PR DESCRIPTION
**Pull request by heavenshell**
Add [graphql-language-service-cli](https://github.com/graphql/graphiql/tree/main/packages/graphql-language-service-cli#readme) for GraphQL.

## Screencast
https://user-images.githubusercontent.com/56591/107531680-6e399780-6c00-11eb-8dd0-0fe0e225c263.mp4

## How to setup

- `:LspInstallServer graphql-language-server`
- Add `.graphqlrc` file to your project root(`.graphqlrc.{js,json,yml,yaml}` or `.graphqlrc.js`).
- `.graphqlrc` should include schema file path
e.g.
```
{
  "schema": "./schema.graphql"
}
```
- download(or generate) GitHub graphql schema 
  - e.g `curl -O https://raw.githubusercontent.com/octokit/graphql-schema/master/schema.graphql` 
- Edit sample.graphql

## Help wanted

I'm not familier with Windows cmd.
Please review `installer/install-graphql-language-server.cmd` file 🙇

### CODE REVIEW

File: installer/install-graphql-language-server.cmd [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+	echo {"name":""}>package.json
+)
+
+call npm install "graphql"
+call npm install "graphql-language-service-cli"
+
+echo @echo off ^
+
+call %%~dp0\node_modules\.bin\graphql-lsp %%* graphql-language-server
```
Comment (heavenshell) [inline installer/install-graphql-language-server.cmd:14]: I want to do `ln -s "./node_modules/.bin/graphql-lsp" graphql-language-server`.
But I'm not shure this is correct...

Comment (mattn) [inline installer/install-graphql-language-server.cmd:14]: Thanks. Please do
```
echo @echo off ^

call %%~dp0\node_modules\.bin\graphql-lsp %%* ^

> graphql-language-server.cmd
```

Comment (heavenshell) [reply installer/install-graphql-language-server.cmd]: https://github.com/mattn/vim-lsp-settings/pull/389/commits/41c2b0c2accdc14455516b244cf9eb4ee9016220
Fixed. 
PTAL 🙏

**Review by mattn — APPROVED** [review]

**Comment by mattn** [comment]
Thank you

**Pull request merged**